sirepo/template/elegant.py

```diff
@@ -1453,3 +1456,23 @@ def _sdds_init():
     _x = getattr(_s, "SDDS_LONGDOUBLE", None)
     _SDDS_DOUBLE_TYPES = [_s.SDDS_DOUBLE, _s.SDDS_FLOAT] + ([_x] if _x else [])
     _SDDS_STRING_TYPE = _s.SDDS_STRING
+
+
+def stateless_compute_log_to_html(data):
```

Comment:
This is a stateful_compute, because it refers to a specific run dir which must have "state" (the log file). 

Here's where it is explained: https://github.com/radiasoft/sirepo/issues/2201#issuecomment-831591025

@e-carlin is there documentation?

Reply:
I think that is an analysis job. statefulCompute is just sirepo-data.json (doesn't go to NERSC for example)

https://github.com/radiasoft/sirepo/wiki/Interacting-with-Simulations-Architecture